test/html_json.test.js

```diff
@@ -13,21 +13,24 @@
 /* eslint-env mocha */
 
 const assert = require('assert');
-const path = require('path');
+const nock = require('nock');
+const { basename, resolve } = require('path');
```

Comment:
I don't like this style...it the `resolve` quite ambigous and when looking at the code below, you have to lookup what resolve actually does.

Reply:
Well, it's a test case, and a small one. To me, `basename` and `resolve` in combination with paths (as in  `resolve(specsDir, 'blog', 'helix-query.yaml')`) is rather obvious.

Reply:
but you had to rename the `resolve` in the promise.... I just think it's not a good style. maybe in this case it's kind of ok, but in general, it's rather confusing, especially when looking at diffs and for people who don't know the code.